subprojects/file-collections/src/test/groovy/org/gradle/api/internal/file/FilePropertyTest.groovy

```diff
@@ -33,7 +33,17 @@ class FilePropertyTest extends FileSystemPropertySpec<RegularFile> {
 
     @Override
     RegularFile someOtherValue() {
-        return baseDir.file("dir2").get()
+        return baseDir.file("other1").get()
```

Comment:
@adammurdoch is the intention here that all methods return a file named `other1` or were they meant to be `other1`, `other2` and `other3`? I can't tell from context.

Reply:
The contract from the superclass is that each test class' `someValue()`, `someOtherValue()` etc each return different values, where "different" means `!=`. Which, of course, isn't happening in this class. Thanks for spotting this, I'll fix it up.